lib/rust/ensogl/component/list-editor/src/lib.rs

```diff
@@ -474,17 +495,26 @@ impl<T: display::Object + frp::node::Data> ListEditor<T> {
             init_drag_not_disabled <- init_drag.gate_not(&drag_disabled);
             is_dragging <- bool(&on_up, &init_drag_not_disabled).on_change();
             drag_diff <- pos_diff.gate(&is_dragging);
+
+            status <- bool(&on_up, &drag_diff).on_change();
+            start <- status.on_true();
+            target_on_start <- target.sample(&start);
+            let on_item_removed = &frp.private.output.on_item_removed;
+            eval target_on_start([model, on_item_removed] (t) {
+                if let Some((index, item)) = model.borrow_mut().start_item_drag(t) {
+                    on_item_removed.emit(Response::gui((index, Rc::new(item).downgrade())));
+                }
+            });
         }
-        (is_dragging, drag_diff)
+        (status, drag_diff)
     }
 
     /// Implementation of item trashing logic. See docs of this crate to learn more.
```

Comment:
This docs should explain return values. Also, the terms in docs should be consistent - here we say about "trashing", but the crates docs (where the user is sent) says about "removing".

To make life easier you may make a link like `See [docs of this crate](crate) ...`